tests/build_local_test_elfs.sh

```diff
@@ -5,39 +5,72 @@ NANOS_SDK=$NANOS_SDK
 NANOX_SDK=$NANOX_SDK
 APP_ETHEREUM=$APP_ETHEREUM
 
-# create elfs folder if it doesn't exist
-mkdir -p elfs
+set -e
 
-# move to repo's root to build apps
-cd ..
+build_plugin() {
+    # arguments: <SDK letter>
+    echo "** Building app-plugin for Nano $1..."
+    local target="tests/elfs/plugin_nano${1,,}.elf"
+    if [ $1 == "S" ];
```

Comment:
[ShellCheck](https://www.shellcheck.net/) detects several problems with this file.

Can you review them and fix them?